core/src/main/java/org/elasticsearch/node/service/NodeService.java

```diff
@@ -135,7 +135,8 @@ public NodeInfo info() {
                 transportService.info(),
                 httpServer == null ? null : httpServer.info(),
                 pluginService == null ? null : pluginService.info(),
-                ingestService == null ? null : ingestService.info()
+                ingestService == null ? null : ingestService.info(),
+                indicesService == null ? null : indicesService.getTotalIndexingBufferBytes()
```

Comment:
Woops, sorry, missed this one: I am not sure?  Guice invokes this constructor right?  I was just being defensive here.  If you are sure `indicesService` cannot be `null`, I'll remove the null check ...

Hmm though it does look like at least one test invokes the constructor directly itself, and passes null for `indicesService`.  Seems best to just remain defensive?